apps/sms/views/inbox/js/inbox.js

```diff
@@ -2,9 +2,9 @@
 /* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
 
 /*global Template, Utils, Threads, Contacts, Threads,
-         WaitingScreen, MessageManager, TimeHeaders,
```

Comment:
anyway we could be smarter about "repeat" time headers; currently we refresh them every minute, just to check whether we changed the day... Likely we should do this only for the relative and day headers (today, yesterday, monday, etc) and check only once a day.